sonar-python-plugin/src/test/java/org/sonar/plugins/python/PythonSensorTest.java

```diff
@@ -1113,7 +1113,7 @@ void write_cpd_tokens_to_cache() throws IOException {
 
     assertThat(writeCache.getData().keySet()).containsExactlyInAnyOrder(
       "python:cache_version", "python:files", "python:descriptors:moduleKey:pass.py", "python:imports:moduleKey:pass.py",
-      "python:cpd:data:moduleKey:pass.py", "python:cpd:stringTable:moduleKey:pass.py", "python:content_hashes:moduleKey:pass.py");
+      "python:cpd:data:moduleKey:pass.py", "python:cpd:stringTable:moduleKey:pass.py", "python:content_hashes:moduleKey:pass.py", "python:typeshed_modules");
```

Comment:
I'm not sure I understand why this change is necessary?

Reply:
It's because now every project will look into Typeshed for at least Django.